loader/loader.go

```diff
@@ -968,11 +975,14 @@ func fetchMatchedLiteral(ctx *tcontext.Context, router *router.Table, schema, ta
 func (l *Loader) restoreData(ctx context.Context) error {
 	begin := time.Now()
 
-	conn, err := createConn(l.cfg)
+	db, conn, err := createConn(ctx, l.cfg)
```

Comment:
It is better to change `db` to another name, as `db` is also used in L991

Reply:
ok, change to ｀baseDB｀